ci/lint/format.sh

```diff
@@ -1,20 +1,16 @@
 #!/usr/bin/env bash
-# Black + Clang formatter (if installed). This script formats all changed files from the last mergebase.
+# Clang formatter (if installed). This script formats all changed files from the last mergebase.
 # You are encouraged to run this locally before pushing changes for review.
 
 # Cause the script to exit if a single command fails
 set -euo pipefail
 
-BLACK_VERSION_REQUIRED="22.10.0"
```

Comment:
We still need all BLACK related code in this file?

Reply:
I've mentioned this in the PR description. `black` is removed from `format.sh` because it is already run in `pre-commit` in CI, so there is no need to run it again in `format.sh`.